java/benchmarks/src/main/java/glide/benchmarks/clients/glide/GlideAsyncClient.java

```diff
@@ -0,0 +1,61 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.benchmarks.clients.glide;
+
+import glide.api.RedisClient;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import glide.benchmarks.clients.AsyncClient;
+import glide.benchmarks.utils.ConnectionSettings;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+
+/** A Glide client with async capabilities */
+public class GlideAsyncClient implements AsyncClient<String> {
+    private RedisClient redisClient;
+
+    @Override
+    public void connectToRedis(ConnectionSettings connectionSettings) {
+        if (connectionSettings.clusterMode) {
+            throw new RuntimeException("Use client GlideAsyncClusterClient");
+        }
+        RedisClientConfiguration config =
+                RedisClientConfiguration.builder()
+                        .address(
+                                NodeAddress.builder()
+                                        .host(connectionSettings.host)
+                                        .port(connectionSettings.port)
+                                        .build())
+                        .useTLS(connectionSettings.useSsl)
+                        .build();
+
+        try {
+            redisClient = RedisClient.CreateClient(config).get();
```

Comment:
Use timeout to avoid hang on error
```suggestion
            redisClient = RedisClient.CreateClient(config).get(10, SECONDS);
```

Reply:
@Yury-Fridlyand if this is the recommended setting, why doesn't `CreateClient` throw a timeout exception itself?

Reply:
If server is unreachable or something wrong in the configuration (issues in network, CI or whatever), `get` call would block the app for eternity.

Reply:
+ timeout exception is thrown only when you specify the timeout for the `get`.